contrib/jenkins/test_walkthrough.sh

```diff
@@ -179,15 +179,15 @@ kubectl --context=service-catalog create -f "${ROOT}/contrib/examples/walkthroug
   || error_exit 'Error when creating ups-binding.'
 
 wait_for_expected_output -e 'InjectedBindResult' \
-  kubectl --context=service-catalog get bindings -n test-ns ups-binding -o yaml \
+  kubectl --context=service-catalog get serviceinstancecredentials -n test-ns ups-binding -o yaml \
```

Comment:
We'll also want to change "ups-binding" to "ups-instance-credential". Fine in a follow-up.